src/core/io.c

```diff
@@ -36,6 +36,7 @@ typedef struct IOFile IOFile;
 struct IOFile {
     FILE *file;
     int flags;
+    char * buf;
```

Comment:
There is a code formatter script that can be run in the tools directory.

Reply:
Done.